package.json

```diff
@@ -70,7 +70,7 @@
     "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
-    "@emotion/core": "^10.0.5",
+    "@emotion/core": "^0.13.1",
```

Comment:
I was able to run storybook with `"@emotion/core": "^10.0.5"` after cleaning the whole cache files and running `yarn install`, so I don't think this dependency change is needed here.

Reply:
Interesting. I am using `npm`, I wonder if there could possibly be a difference there. I'll try yarn and see what happens. I'll also remove this change.

Reply:
Did you delete `package-lock.json` as well?

Reply:
yes, I deleted `package-lock.json`as well.

Reply:
well i guess most of the team (if not all) are using yarn that's why `package-lock.json` was deleted (if nobody is using npm install then it would get pretty outdated no). you have a `yarn.lock` for the same purposes. 
(not trying to debate about yarn vs npm here just stating what i understand)

Reply:
Oh, that makes sense. I'm ok with using `yarn` when contributing. I know every team has their own tools.